wagtail/admin/templates/wagtailadmin/pages/confirm_delete.html

```diff
@@ -68,6 +68,20 @@
 
         <form action="{% url 'wagtailadmin_pages:delete' page.id %}" method="POST">
             {% csrf_token %}
+            {% if confirm_before_delete %}
+                <p class="status-msg warning">
+                    {% blocktrans with total_pages=descendant_count|add:1 %}
```

Comment:
Please can you add the `trimmed` option to the blocktrans tags? This ensures that the leading/trailing whitespace doesn't end up in the translation strings (which makes the translation process more awkward for translators).
```suggestion
                    {% blocktrans trimmed with total_pages=descendant_count|add:1 %}
```